test/common.bash

```diff
@@ -77,6 +79,14 @@ prerequisites_ok () {
     fi
 }
 
+revert_crayify () {
+    if [[ $ch_cray ]]; then
+        $ch_mpirun_node ch-tar2dir "${ch_tardir}/${ch_tag}.tar.gz" "$ch_imgdir"
+    else
+        skip 'image mpi not modified'
+    fi
+}
+
```

Comment:
If we are going to use an unpacking image function, I feel like it should be generic, e.g., `unpack_image ()` or perhaps `revert_image ()`.

That said, didn't we want to re-order the guest and host launches to avoid using such a function? `run_auto.bats` will unpack the image each time the test suite is executed.

FWIIW, I have suggested adding a function like this in this past. One of the benefits is that we can target portions of the test suite, e.g., `make test-test` without having to manually unpack the images or run `make test-run` prior. However, it is reasonable to argue that a quality of life change, such as this, doesn't really justify deviating from the standard test suite behavior.